File: VB/WpfApplication1/WizardCustomization/CustomWelcomePage.vb
Imports Microsoft.VisualBasic
Imports DevExpress.Data.XtraReports.Wizard
Imports DevExpress.Data.XtraReports.Wizard.Presenters
Imports DevExpress.Data.XtraReports.Wizard.Views
Imports System
' ...

Namespace WpfApplication1.WizardCustomization
	Friend Class CustomWelcomePage
		Inherits WelcomePage(Of ReportModel)
		Public Sub New(ByVal view As IWelcomePageView)
			MyBase.New(view)
		End Sub

		Public Overrides Function GetNextPageType() As Type
			Return GetType(CustomSelectDataSourcePage)
		End Function
	End Class
End Namespace

File: CS/WpfApplication1/WizardCustomization/CustomWelcomePage.cs
using DevExpress.Data.XtraReports.Wizard;
using DevExpress.Data.XtraReports.Wizard.Presenters;
using DevExpress.Data.XtraReports.Wizard.Views;
using System;
// ...

namespace WpfApplication1.WizardCustomization {
    class CustomWelcomePage : WelcomePage<ReportModel> {
        public CustomWelcomePage(IWelcomePageView view) : base(view) { }

        public override Type GetNextPageType() {
            return typeof(CustomSelectDataSourcePage);
        }
    }
}
